openmp-extras/aomp-flang-decouple-out-dir-and-rocm-install.patch

```diff
@@ -0,0 +1,57 @@
+diff --color -urN aomp-rocm-4.5.0-unpatched/bin/build_flang_runtime.sh aomp-rocm-4.5.0/bin/build_flang_runtime.sh
```

Comment:
Great work! I always had issues with this part and it stopped me from working on the package. Thank you.

Reply:
Yeah the whole "build and install step-by-step directly into the filesystem" approach AMD is taking is a bit awkward